assets_src/config/gameplay/structures/gold_camp.yaml

```diff
@@ -21,7 +21,7 @@ unitClasses:
     tags:
       - structure
     verbs:
-      - unload
+      - unload_two
```

Comment:
out of curiosity, have you tried unloading manually in game. I had troubles creating a new unload verb because unload was handled differently by game (an unload is generated for each unit loaded)

Reply:
Using the game's unloading could mean that the AI would unload at any given point, even if there was still gold to mine in the camp or they could never unload, even if there is no gold in the camp. unload_two gives me control over how the AI unloads

Reply:
Fixed